### ISSUE TITLE
Multiple issues with PPPoE WAN interface

### ISSUE THREAD

**Issue by telnetdoogie** [telnetdoogie/UDMP-ipv6#6]
```shell
root@Dream-Machine-Special-Edition:/data/ipv6# WAN_IFACE=$(ip route get 8.8.8.8 | awk '{ printf $5 }')
root@Dream-Machine-Special-Edition:/data/ipv6# echo $WAN_IFACE
201.ppp0
```
updated script incorrectly identified ppp0 interface, and firewall rules appear to be removed on the UDMPSE...

@DJBenson can you add any more details here?

**Comment by telnetdoogie** [comment]
updated script should resolve the resolution of the ppp0 interface, but we still have to figure out the ip6tables losses.

**Comment by telnetdoogie** [comment]
@djbenson [this](https://community.ui.com/questions/Enabled-IPv6-now-my-UDM-Pros-logs-are-getting-filled-with-Multipath-routes-not-supported-errors/9bc5d01d-6a56-4cda-9e13-887588a3383c)  looks like your issue…

**Comment by telnetdoogie** [comment]
@DJBenson I suspect that your issue is unrelated to the firewall script...

Can you try to ONLY setup the tunnel with `enable-he-ipv6.sh` and NOT run `configure-he-ipv-chains.sh` (you'll have to remove the cron job as well) and re-check your logs to see if you're still having those `Multipath routes not supported` errors? I suspect they'll still happen so there may be an issue with the tunnel before we get to firewall rules.

**Comment by DJBenson** [comment]
> @DJBenson [this](https://community.ui.com/questions/Enabled-IPv6-now-my-UDM-Pros-logs-are-getting-filled-with-Multipath-routes-not-supported-errors/9bc5d01d-6a56-4cda-9e13-887588a3383c) looks like your issue…

Yeah I found the same, sadly unresolved. I'll raise it with Ubiquiti but I need to find a way to replicate it without any hackery as they won't touch it with a bargepole if they know there's unsupported changes to the UDM-SE.

**Comment by telnetdoogie** [comment]
Yeah I don't think they'll even support using an HE tunnel

**Comment by telnetdoogie** [comment]
sounds like a custom kernel may be required

**Comment by telnetdoogie** [comment]
for posterity:

Issues in log:

```
2023-05-12T11:59:17+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:18+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:19+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:21+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:22+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:24+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:25+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:27+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:28+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:30+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:31+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:33+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:34+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:36+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:37+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:38+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:40+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:45+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:46+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Trying to migrate config due to inconsistency (invalid config .versionDetail: must be between 9 and 9: .interfaces) to resolve following issue: invalid config: configuration syntax is invalid: IDSIPSSignaturesMode must be an object: .services.idsIps.signatures
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Starting config .versionFormat 'v2' migration for /tmp/udapi-fastapply-82b6_d6fc_82b7_c253.cfg.tmp
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/filter from 1 to 2
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/filter from 2 to 3
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/mangle from 1 to 2
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/mangle from 2 to 3
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/mangle from 3 to 4
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/nat from 1 to 2
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/nat from 2 to 3
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/pbr from 1 to 2
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/pbr from 2 to 3
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.firewall/pbr from 3 to 4
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.interfaces from 8 to 9
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.qos/ip from 1 to 2
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.routes/ospf from 2 to 3
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.routes/ospf/areas from 2 to 3
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.services/ddns from 1 to 2
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.services/dnsForwarder from 2 to 3
2023-05-12T11:59:51+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.services/idsIps from 1 to 2
2023-05-12T11:59:52+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.services/igmpSnooping from 1 to 2
2023-05-12T11:59:52+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.services/l2tpServer from 1 to 2
2023-05-12T11:59:52+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.services/wanFailover from 4 to 5
2023-05-12T11:59:52+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.system from 1 to 2
2023-05-12T11:59:52+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.system from 2 to 3
2023-05-12T11:59:52+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.system from 3 to 4
2023-05-12T11:59:52+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/ipsec/site-to-site from 1 to 2
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/ipsec/site-to-site from 2 to 3
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/peers from 1 to 2
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/peers from 2 to 3
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/peers from 3 to 4
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.qos from 1 to 2
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/raws from 2 to 3
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/clients from 0 to 1
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/clients from 1 to 2
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/site-to-sites from 1 to 2
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/site-to-sites from 2 to 3
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/servers from 1 to 2
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.services/dohProxy from 0 to 1
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.services/stunnel from 0 to 1
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrating config .versionDetail.routes/access-lists from 0 to 1
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Finished config .versionFormat 'v2' migration of /tmp/udapi-fastapply-82b6_d6fc_82b7_c253.cfg.tmp
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: config-migrate-helper: Migrated config is valid
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service: vvv Apply new configuration
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                    [interfaces]: configuring
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                 [routes/static]: disabling
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                   [routes/ospf]: disabling
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                   [peripherals]: disabling
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                        [system]: configuring
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 245 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 246 nexthops for route ::/0 dev he-ipv6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *      [services/radius-profiles]: configuring
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                      [services]: configuring
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-redirector-service:         +(services):    Keep running service redirector
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-arp-inspection-service:     +(services):    Keep stopped service arpInspection
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_IoT_br100_192-168-100-0-24
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Security_br101_192-168-101-0-24
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Guest_br102_192-168-102-0-24
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Management_br103_192-168-103-0-24
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Default_br0_192-168-1-0-24
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Default_br0_192-168-1-0-24_IPV6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Management_br103_192-168-103-0-24_IPV6
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-dns-forwarder-service:      +(services):    Keep running service dnsForwarder
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-dpi-service:                +(services): Restart running service dpi
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-geoip-filtering-service:    +(services):    Keep running service geoipFiltering
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-ids-ips-service:            +(services):    Keep running service idsIps
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-igmp-snooping-snoopd:       +(services):    Keep running service igmpSnooping
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-l2tp-server:                +(services):    Keep running service l2tpServer
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-lldp:                       +(services):    Keep running service lldp
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-mdns-service:               +(services):    Keep running service mdns
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-ntp-client-timesyncd:       +(services):    Keep running service ntpClient
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-radius-server-service:      +(services):    Keep running service radiusServer
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-system-log-syslog-ng:       +(services):    Keep stopped service systemLog
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-unifi-network:              +(services):    Keep running service unifiNetwork
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-utm-service:                +(services):    Keep running service utm
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-wan-failover:               +(services):    Keep running service wanFailover
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: svc-wifiman:                    +(services):    Keep running service wifiman
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                 [firewall/sets]: configuring
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                  [firewall/nat]: configuring
2023-05-12T11:59:53+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *               [firewall/filter]: configuring
2023-05-12T11:59:54+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *               [firewall/mangle]: configuring
2023-05-12T11:59:54+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                           [qos]: configuring
2023-05-12T11:59:54+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *                         [vlans]: disabling
2023-05-12T11:59:54+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *        [bridge-firewall/broute]: configuring
2023-05-12T11:59:54+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service:  *           [bridge-firewall/nat]: configuring
2023-05-12T11:59:54+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: service: ^^^ Apply new configuration done
2023-05-12T11:59:54+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: process: Got process exit event for process redirector
2023-05-12T11:59:55+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1333]: netlink: Multipath routes not supported, got 246 nexthops for route ::/0 dev he-ipv6```

**Comment by telnetdoogie** [comment]
@DJBenson can you clarify, this issue happens only after modifying `ip6tables` or it happens just when having the tunnel connected? If you have the tunnel but no firewall modifications do you still get errors?

Also, can you confirm that you have ipv6 `DISABLED` on all of your WAN / Internet definitions in the UI?

**Comment by DJBenson** [comment]
> @DJBenson can you clarify, this issue happens only after modifying `ip6tables` or it happens just when having the tunnel connected? If you have the tunnel but no firewall modifications do you still get errors?
> 
> 
> 
> Also, can you confirm that you have ipv6 `DISABLED` on all of your WAN / Internet definitions in the UI?
> 
> 

I'll have a play later and come back to you. 

I'm going to back out the custom changes and see if simply enabling the IPv6 interface on my main VLAN causes the issue. I'll then install the scripts again and take it from there.

Given that other report was native IPv6 rather than a tunnel it may not happen until the tunnel is enabled but I'll play around with a few scenarios and document the outcome.

**Comment by telnetdoogie** [comment]
@DJBenson if you wouldn't mind as well running `ifconfig | grep -i 'eth[0-9]\|br[0-9]\|ppp'` and posting the output? Since I don't have PPPoE it's hard for me to know how the interfaces look. I'm especially interested whether you have a VLAN interface on your WAN (like `ppp0.30` or something)

**Comment by DJBenson** [comment]
> @DJBenson if you wouldn't mind as well running `ifconfig | grep -i 'eth[0-9]\|br[0-9]\|ppp'` and posting the output? Since I don't have PPPoE it's hard for me to know how the interfaces look. I'm especially interested whether you have a VLAN interface on your WAN (like `ppp0.30` or something)

```
root@Dream-Machine-Special-Edition:~# ifconfig | grep -i 'eth[0-9]\|br[0-9]\|ppp'
br0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
br100: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
br101: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
br102: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
br103: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth3: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth4: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth6: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth7: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth8: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth9: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
eth10: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth10.100: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth10.101: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth10.102: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth10.103: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
ppp0: flags=4305<UP,POINTOPOINT,RUNNING,NOARP,MULTICAST>  mtu 1492
        ppp  txqueuelen 3  (Point-to-Point Protocol)
```

**Comment by DJBenson** [comment]
Freshly rebooted UDM, scripts disabled (simply renamed to prevent execution), IPv6 interface disabled:
- No errors

IPv6 interface enabled on primary VLAN:
- Invalid config errors appear
- No sign of the multipath errors
```
2023-05-15T11:38:40+01:00 Dream-Machine-Special-Edition syswrapper[13787]: [apply-config] using fast apply
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Trying to migrate config due to inconsistency (invalid config .versionDetail: must be between 9 and 9: .interfaces) to resolve following issue: invalid config: configuration syntax is invalid: IDSIPSSignaturesMode must be an object: .services.idsIps.signatures
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Starting config .versionFormat 'v2' migration for /tmp/udapi-fastapply-c78a_b7ba_41bf_af2b.cfg.tmp
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/filter from 1 to 2
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/filter from 2 to 3
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/mangle from 1 to 2
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/mangle from 2 to 3
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/mangle from 3 to 4
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/nat from 1 to 2
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/nat from 2 to 3
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/pbr from 1 to 2
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/pbr from 2 to 3
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/pbr from 3 to 4
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.interfaces from 8 to 9
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.qos/ip from 1 to 2
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.routes/ospf from 2 to 3
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.routes/ospf/areas from 2 to 3
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/ddns from 1 to 2
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/dnsForwarder from 2 to 3
2023-05-15T11:38:42+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/idsIps from 1 to 2
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/igmpSnooping from 1 to 2
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/l2tpServer from 1 to 2
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/wanFailover from 4 to 5
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.system from 1 to 2
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.system from 2 to 3
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.system from 3 to 4
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/ipsec/site-to-site from 1 to 2
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/ipsec/site-to-site from 2 to 3
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/peers from 1 to 2
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/peers from 2 to 3
2023-05-15T11:38:43+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/peers from 3 to 4
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.qos from 1 to 2
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/raws from 2 to 3
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/clients from 0 to 1
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/clients from 1 to 2
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/site-to-sites from 1 to 2
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/site-to-sites from 2 to 3
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/servers from 1 to 2
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/dohProxy from 0 to 1
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/stunnel from 0 to 1
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.routes/access-lists from 0 to 1
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Finished config .versionFormat 'v2' migration of /tmp/udapi-fastapply-c78a_b7ba_41bf_af2b.cfg.tmp
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrated config is valid
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service: vvv Apply new configuration
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                    [interfaces]: configuring
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                 [routes/static]: disabling
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                   [routes/ospf]: disabling
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                   [peripherals]: disabling
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                        [system]: configuring
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *      [services/radius-profiles]: configuring
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                      [services]: configuring
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-redirector-service:         +(services):    Keep running service redirector
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-arp-inspection-service:     +(services):    Keep stopped service arpInspection
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_IoT_br100_192-168-100-0-24
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Security_br101_192-168-101-0-24
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Guest_br102_192-168-102-0-24
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Management_br103_192-168-103-0-24
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Default_br0_192-168-1-0-24
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Default_br0_192-168-1-0-24_IPV6
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Management_br103_192-168-103-0-24_IPV6
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dns-forwarder-service:      +(services):    Keep running service dnsForwarder
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dpi-service:                +(services): Restart running service dpi
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-ids-ips-service:            +(services):    Keep running service idsIps
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-igmp-snooping-snoopd:       +(services):    Keep running service igmpSnooping
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-l2tp-server:                +(services):    Keep running service l2tpServer
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-lldp:                       +(services):    Keep running service lldp
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-mdns-service:               +(services):    Keep running service mdns
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-ntp-client-timesyncd:       +(services):    Keep running service ntpClient
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-radius-server-service:      +(services):    Keep running service radiusServer
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-system-log-syslog-ng:       +(services):    Keep stopped service systemLog
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-unifi-network:              +(services):    Keep running service unifiNetwork
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-utm-service:                +(services):    Keep running service utm
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-wan-failover:               +(services):    Keep running service wanFailover
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-wifiman:                    +(services):    Keep running service wifiman
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                 [firewall/sets]: configuring
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                  [firewall/nat]: configuring
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *               [firewall/filter]: configuring
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *               [firewall/mangle]: configuring
2023-05-15T11:38:44+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                           [qos]: configuring
2023-05-15T11:38:45+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                         [vlans]: disabling
2023-05-15T11:38:45+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *        [bridge-firewall/broute]: configuring
2023-05-15T11:38:45+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *           [bridge-firewall/nat]: configuring
2023-05-15T11:38:45+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service: ^^^ Apply new configuration done
2023-05-15T11:38:45+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: process: Got process exit event for process redirector
2023-05-15T11:38:46+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Trying to migrate config due to inconsistency (invalid config .versionDetail: must be between 9 and 9: .interfaces) to resolve following issue: invalid config: configuration syntax is invalid: IDSIPSSignaturesMode must be an object: .services.idsIps.signatures
2023-05-15T11:38:46+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Starting config .versionFormat 'v2' migration for /tmp/udapi-fastapply-2f57_cdf3_9422_dfdc.cfg.tmp
2023-05-15T11:38:46+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/filter from 1 to 2
2023-05-15T11:38:46+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/filter from 2 to 3
2023-05-15T11:38:46+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/mangle from 1 to 2
2023-05-15T11:38:46+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/mangle from 2 to 3
2023-05-15T11:38:46+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/mangle from 3 to 4
2023-05-15T11:38:46+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/nat from 1 to 2
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/nat from 2 to 3
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/pbr from 1 to 2
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/pbr from 2 to 3
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.firewall/pbr from 3 to 4
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.interfaces from 8 to 9
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.qos/ip from 1 to 2
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.routes/ospf from 2 to 3
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.routes/ospf/areas from 2 to 3
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/ddns from 1 to 2
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/dnsForwarder from 2 to 3
2023-05-15T11:38:47+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/idsIps from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/igmpSnooping from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/l2tpServer from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/wanFailover from 4 to 5
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.system from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.system from 2 to 3
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.system from 3 to 4
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/ipsec/site-to-site from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/ipsec/site-to-site from 2 to 3
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/peers from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/peers from 2 to 3
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/peers from 3 to 4
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.qos from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/openvpn/raws from 2 to 3
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/clients from 0 to 1
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/clients from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/site-to-sites from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/site-to-sites from 2 to 3
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.vpn/wireguard/servers from 1 to 2
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/dohProxy from 0 to 1
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.services/stunnel from 0 to 1
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrating config .versionDetail.routes/access-lists from 0 to 1
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Finished config .versionFormat 'v2' migration of /tmp/udapi-fastapply-2f57_cdf3_9422_dfdc.cfg.tmp
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: config-migrate-helper: Migrated config is valid
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service: vvv Apply new configuration
2023-05-15T11:38:48+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                    [interfaces]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                 [routes/static]: disabling
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                   [routes/ospf]: disabling
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                   [peripherals]: disabling
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                        [system]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *      [services/radius-profiles]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                      [services]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-redirector-service:         +(services):    Keep running service redirector
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-arp-inspection-service:     +(services):    Keep stopped service arpInspection
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_IoT_br100_192-168-100-0-24
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Security_br101_192-168-101-0-24
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Guest_br102_192-168-102-0-24
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Management_br103_192-168-103-0-24
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Default_br0_192-168-1-0-24
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Default_br0_192-168-1-0-24_IPV6
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dhcp-server-service:        +(services):    Keep running service dhcpServers-net_Management_br103_192-168-103-0-24_IPV6
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dns-forwarder-service:      +(services):    Keep running service dnsForwarder
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-dpi-service:                +(services): Restart running service dpi
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-ids-ips-service:            +(services):    Keep running service idsIps
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-igmp-snooping-snoopd:       +(services):    Keep running service igmpSnooping
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-l2tp-server:                +(services):    Keep running service l2tpServer
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-lldp:                       +(services):    Keep running service lldp
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-mdns-service:               +(services):    Keep running service mdns
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-ntp-client-timesyncd:       +(services):    Keep running service ntpClient
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-radius-server-service:      +(services):    Keep running service radiusServer
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-system-log-syslog-ng:       +(services):    Keep stopped service systemLog
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-unifi-network:              +(services):    Keep running service unifiNetwork
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-utm-service:                +(services):    Keep running service utm
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-wan-failover:               +(services):    Keep running service wanFailover
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: svc-wifiman:                    +(services):    Keep running service wifiman
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                 [firewall/sets]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                  [firewall/nat]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *               [firewall/filter]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *               [firewall/mangle]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                           [qos]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *                         [vlans]: disabling
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *        [bridge-firewall/broute]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service:  *           [bridge-firewall/nat]: configuring
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: service: ^^^ Apply new configuration done
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: netlink: Unable to resolve hostname for neighbor 2001:470:1d26:1::50: Host not found (non-authoritative), try again later
2023-05-15T11:38:49+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: process: Got process exit event for process redirector
2023-05-15T11:39:06+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1339]: netlink: Unable to resolve hostname for neighbor 2001:470:1d26:1::788: Host not found (non-authoritative), try again later
```

Executed configure-he-ipv6-chains.sh:
- No additional errors (invalid config errors continue)

Executed enable-he-ipv6.sh:
- No additional errors - conversely the logs have quietened down.

**Comment by DJBenson** [comment]
I haven't seen the multipath errors today but the invalid config issue is very active - whatever is going on, I assume that is what is wiping the rules set up by your script as it seems to be reloading iptables/iptables6 every few seconds.

The last update on that ticket was several months ago and I'm running the EA build of the applications. The frustrating thing is I can't prove whether it's core unifi-os at fault because I don't have a native IPv6 connection to use.

**Comment by DJBenson** [comment]
So I retraced all the steps and noticed something in the Caveat Emptor section, disabling the IPv6 connection on the WAN interface. I hadn't touched these settings this time round but must have configured it previously. I wonder if that was the source of the multipath issue - anyway I've not seen that again since.

The invalid config issue persists.

**Comment by DJBenson** [comment]
[Issue raised via UI community](https://community.ui.com/questions/UDM-SE-Constant-reloading-of-firewall-rules-due-to-invalid-config/73fa0dc0-a621-4b4c-917c-b3fb38024d2e)

**Comment by telnetdoogie** [comment]
@DJBenson 

> IPv6 interface enabled on primary VLAN:

When you say this, you're talking about enabling IPv6 on your LAN (internal) Network versus the WAN is that correct?

**Comment by telnetdoogie** [comment]
FYI @DJBenson thanks for all the discovery and troubleshooting. Super appreciated!

**Comment by telnetdoogie** [comment]
Proposed / Potential Solution here, possibly a rule migration problem from previous versions?? Not sure though, haven't seen any confirmation aside from that one comment:

https://community.ui.com/questions/UDR-config-migrate-helper-runs-every-time-I-changes-a-configuration/ef0eee88-3b19-4e01-b67e-2a4ad487cf42#answer/bd7f9fc7-0f37-49bb-9706-738fdf687838

**Comment by DJBenson** [comment]
> @DJBenson
> 
> > IPv6 interface enabled on primary VLAN:
> 
> When you say this, you're talking about enabling IPv6 on your LAN (internal) Network versus the WAN is that correct?

Yes indeed. My network is segmented into 4 VLANs with VLAN1 being my primary network, it's on this network I am experimenting with IPv6 before enabling it across all my VLANs. As above I messed up a bit as I had some leftover config on my WAN interface but that's disabled now (and still no multipath errors so I think that's where those were originating from).

**Comment by DJBenson** [comment]
> FYI @DJBenson thanks for all the discovery and troubleshooting. Super appreciated!

No - thank you - this issue is probably out of scope for you but I've appreciated your help.

**Comment by DJBenson** [comment]
> Proposed / Potential Solution here, possibly a rule migration problem from previous versions?? Not sure though, haven't seen any confirmation aside from that one comment:
> 
> https://community.ui.com/questions/UDR-config-migrate-helper-runs-every-time-I-changes-a-configuration/ef0eee88-3b19-4e01-b67e-2a4ad487cf42#answer/bd7f9fc7-0f37-49bb-9706-738fdf687838

Quite possibly - I've migrated from a UDM-PRO to a UDM-SE and I run EA code (for my sins) so could have some old config floating around.

I'm reluctant to start pulling rules out as I run a smart home with impatient kids and wife and if I break something they'll not be happy 😉 I'll try disabling some rules I know aren't critical. It's a shame Ubiquiti don't have a method like pfSense where you can backup parts of the system, so only the firewall rules...

**Comment by telnetdoogie** [comment]
@DJBenson if you do try it, I think you could just do the ipv6 rules only. I'd start there. I'd have a similar concern about starting to break down the others... I also run EA and haven't done any kind of hard reset between my first install and now, despite all the migrations.

**Comment by DJBenson** [comment]
I only have two test rules on my IPv6 interface, one to block everything and one to allow a test port (443 in this case).

What is enabled is the DPI stuff which I assume is active on all interfaces - the logs reference something relating to DPI but disabling it 1) doesn't seem to make a difference and 2) isn't something I'd want to do in anger.

**Comment by DJBenson** [comment]
I've managed to capture the file which is being applied - it is of course full of sensitive data but I wondered if you had any experience of this file and where to look for issues or even if there's anything useful in this file (the message log refers to the interfaces node having the wrong config version, it mentions v2 but then later says it's v8 and will be migrated to v9 and the file I captured shows the config version at v9 after migration).

```
2023-05-16T13:11:32+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1343]: config-migrate-helper: Trying to migrate config due to inconsistency (invalid config .versionDetail: must be between 9 and 9: .interfaces) to resolve following issue: invalid config: configuration syntax is invalid: must be between 9 and 9: .versionDetail.interfaces
2023-05-16T13:11:32+01:00 Dream-Machine-Special-Edition ubios-udapi-server[1343]: config-migrate-helper: Starting config .versionFormat 'v2' migration for /tmp/udapi-fastapply-5617_12ae_c660_c68a.cfg.tmp
```

![image](https://github.com/telnetdoogie/UDMP-ipv6/assets/1013909/bfb029fb-f2b1-4588-8c99-1baec05a7007)

**Comment by telnetdoogie** [comment]
I had been wondering if you were able to view that file, actually, based on the log entries... so I'm glad you poked arouind at it... nothing looks wrong there to me, but `must be between 9 and 9` seems to suggest an underlying problem versus a problem with the file itself, that doesn't make much sense to me... 

Found another reference to it on the 3.0.20 firmware page which is pretty recent... [this](https://community.ui.com/releases/UniFi-OS-Dream-Machines-3-0-20/1e3e2424-3ed9-411b-b872-a27bb30eb345#comment/2d30be91-ff72-4ae5-8823-7d8a53471ad0) seems to suggest that this 'migration' would happen once, though, and then be done, so the error appears at upgrade time versus repeating over and over... What OS are you running right now?

**Comment by telnetdoogie** [comment]
Also, it might be worth deleting ipv4 rules as well, I guess, based on what I'm seeing others say... this is starting to smell more like a general upgrade / OS / Network version issue versus ipv6 specific.

**Comment by telnetdoogie** [comment]
@DJBenson Any updates?

**Comment by telnetdoogie** [comment]
@DJBenson ever get this resolved?

**Comment by DJBenson** [comment]
Hi @telnetdoogie

I didn't. I kinda abandoned the project because I didn't really want to be redoing all my firewall rules. I suspect a clean install would resolve the issue, my UDM-SE has been through many upgrades including moving from the UDM-Pro so it probably has some cobwebs but I don't have the time at the moment to do a clean install.

**Comment by telnetdoogie** [comment]
Thanks!

I'm gonna close this issue for now, but if you pick things back up again and re-encounter the issue be sure to check back and we can get things going again...

**Comment by DJBenson** [comment]
I know there's quite a bit of time passed but I wanted to just let you know that using the latest version of your script on the latest version of UniFi OS works brilliantly now. No idea what was causing the issues I previously had but it's now working, my clients are slowly getting IPv6 connectivity back and I've managed to enable IPv6 on my self-hosted apps (exposed to the internet using sslh, Docker and, nginxproxymanager and Authelia) and all is good in the world (the last bit is a massive lie but for a different place/time).

Thanks again for your scripts and the time you spent last year trying to get me set up.

**Comment by telnetdoogie** [comment]
Glad things are working! Could have been the PPP modification I made, or just a UDMP glitch.

Appreciate the learnings and the opportunity to improve the scripts!!

**Comment by sielicki** [comment]
It's really unbelievable that Ubiquiti does not support this out of the box. 

Thank you for the scripts.
